File: DSA/akashyadav_590014126/Day_10_Question_1.py
def check_for_strings(s1, s2):
    
    if s1 == s2:
        eq = "Equal"    # if they match equally
    else:
        eq = "Not equal"    # if they doesn't match completely


    if s1.lower() == s2.lower():
        case_eq = "Equal ignoring case" # if they are equal in case insensitive
    else:
        case_eq = "Not equal ignoring case" # if in case insensitive also they are not equal


    if s1 == s2:
        order = "Strings are identical" # if both are equal 
    elif s1 < s2:
        order = f'"{s1}" comes before "{s2}"'
    else:
        order = f'"{s2}" comes before "{s1}"'

    return f"{eq}, {case_eq}, {order}"

#Example
print(check_for_strings("apple", "banana"))

File: DSA/Shubham_590016506/Day_10_Question_1.c
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main() {
    char s1[100], s2[100];
    scanf("%s %s", s1, s2);

    if (strcmp(s1, s2) == 0)
        printf("Equal, Equal ignoring case, Strings are identical\n");
    else {
        printf("Not equal, ");
        int i = 0, f = 1;
        while (s1[i] && s2[i]) {
            if (tolower(s1[i]) != tolower(s2[i])) {
                f = 0;
                break;
            }
            i++;
        }
        if (f && s1[i] == '\0' && s2[i] == '\0')
            printf("Equal ignoring case, ");
        else
            printf("Not equal ignoring case, ");

        if (strcmp(s1, s2) < 0)
            printf("\"%s\" comes before \"%s\"\n", s1, s2);
        else
            printf("\"%s\" comes before \"%s\"\n", s2, s1);
    }
    return 0;
}